src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -93,6 +93,10 @@ Transaction *TimestampOrderingTransactionManager::BeginTransaction() {
   auto eid = EpochManagerFactory::GetInstance().EnterEpoch(begin_cid);
   txn->SetEpochId(eid);
 
+  auto &log_manager = logging::LogManager::GetInstance();
+  log_manager.PrepareLogging();
```

Comment:
I don't think there is any different though... we can talk about this offline...